ID.me WebVerify SDK/IDmeWebVerify.m

```diff
@@ -99,7 +112,15 @@ - (void)verifyUserInViewController:(UIViewController *)externalViewController
                              scope:(NSString *)scope
                    withTokenResult:(IDmeVerifyWebVerifyTokenResults)webVerificationResults {
     NSAssert(self.clientID != nil, @"You should initialize the SDK before making requests. Call IDmeWebVerify.initializeWithClientID:redirectURI");
-    NSURL* url =  [NSURL URLWithString: [self urlStringWithQueryString:[NSString stringWithFormat: IDME_WEB_VERIFY_GET_AUTH_URI, self.clientID, self.redirectURI, scope]]];
+    NSAssert(self.safariViewController == nil, @"There is an authentication flow in progress. You should not call IDmeWebVerify.verifyUserInViewController:scope:withTokenResult until the previous has finished");
```

Comment:
Maybe it would be better to return an error instead of an assertion failure. What do you think?